scheme.html

```diff
@@ -6,7 +6,7 @@
       name="viewport"
       content="initial-scale=1,maximum-scale=1,user-scalable=no"
     />
-    <title>ATIP v2 Scheme Entry</title>
+    <title>ATIP v2 Scheme Sketcher</title>
```

Comment:
Somehow this change feels quite momentous

Reply:
Just something I noticed. Do we have a better name? >_<

Reply:
I don't think so, I just mean that we've been talking for a while about various issues with naming and so on, and rebranding to scheme sketcher. Now it's here!